Spigot-Server-Patches/0544-Add-Player-teleportRelative-and-Player-teleportWitho.patch

```diff
@@ -0,0 +1,51 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Anrza <andrzejrzeczycki314@gmail.com>
+Date: Wed, 15 Jul 2020 12:28:43 +0200
+Subject: [PATCH] Add Player#teleportRelative and
+ Player#teleportWithoutRotation
+
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java b/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
+index b832ece4948323e60fabecdeee5c9051b65c5e8e..1839082cc5e56b201cec722ded7db83324dfcc4e 100644
+--- a/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
++++ b/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
+@@ -18,6 +18,7 @@ import java.net.InetSocketAddress;
+ import java.net.SocketAddress;
+ import java.util.ArrayList;
+ import java.util.Collection;
++import java.util.EnumSet;
```

Comment:
Imports can be in-lined for smaller diff

Reply:
I'm unsure what you mean. Should I put the import in the method instead?

Reply:
Like where you use this import you can just inline the full package instead of importing it in the header
```
java.util.EnumSet.allOf(PacketPlayOutPosition.EnumPlayerTeleportFlags.class)
```
;)